src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -141,7 +141,7 @@ public static OpenLoadFlowParameters load() {
     public String toString() {
         return "OpenLoadFlowParameters(" +
                 "slackBusSelectionMode=" + slackBusSelectionMode +
-                ", slackBusId='" + Objects.requireNonNull(slackBusId, "") + "'" +
+                ", slackBusId='" + slackBusId + "'" +
```

Comment:
The mistake is that Objects.requireNonNull has been used instead of Objects.toString

Reply:
well, `", slackBusId='" + slackBusId + "'"` with a null `slackbusId` gives `, slackBusId='null'`, while your fix gives `, slackBusId=''`